tasks/cloudsight_setup.yml

```diff
@@ -6,7 +6,7 @@
 
 - name: Create ThreatStack Config File
   template:
-    src: templates/config.j2
+    src: config.j2
```

Comment:
What version of Ansible are you on?  Wondering how it relates to this issue.

https://github.com/ansible/ansible/issues/14161